### ISSUE TITLE
Fix S5332 FP: Don't raise http warning for xml namespace

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#4124]
As seen on SonarCloud for S4MSB:
https://sonarcloud.io/project/security_hotspots?id=sonarscanner-msbuild&pullRequest=937&hotspots=AXgCSAIHik2AsAwb1Bpy

HTTP should not trigger the issue when assigned to field/variable/property/argument with "Namespace" in it's name:

```C#
[XmlRoot(ElementName = "SonarProjectConfig", Namespace = "http://www.sonarsource.com/msbuild/analyzer/2021/1")]
public class ProjectConfig
{
}
```

Same FP will happen in #4120 in our repo.
